src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/contentgrid/view/ContentGrid.kt

```diff
@@ -2,32 +2,84 @@ package org.wycliffeassociates.otter.jvm.app.ui.contentgrid.view
 
 import de.jensd.fx.glyphs.materialicons.MaterialIcon
 import de.jensd.fx.glyphs.materialicons.MaterialIconView
+import javafx.beans.binding.Bindings.createBooleanBinding
+import javafx.beans.binding.BooleanExpression
 import javafx.beans.property.Property
 import javafx.event.EventHandler
+import javafx.scene.Node
 import javafx.scene.layout.Priority
 import org.wycliffeassociates.otter.common.data.model.Collection
 import org.wycliffeassociates.otter.common.data.model.Content
+import org.wycliffeassociates.otter.common.data.model.ResourceMetadata
 import org.wycliffeassociates.otter.jvm.app.theme.AppStyles
 import org.wycliffeassociates.otter.jvm.app.theme.AppTheme
 import org.wycliffeassociates.otter.jvm.app.ui.contentgrid.viewmodel.ContentGridViewModel
+import org.wycliffeassociates.otter.jvm.app.ui.contentgrid.viewmodel.ContentTuple
+import org.wycliffeassociates.otter.jvm.app.ui.contentgrid.viewmodel.GroupedContent
 import org.wycliffeassociates.otter.jvm.app.ui.mainscreen.view.MainScreenStyles
+import org.wycliffeassociates.otter.jvm.app.widgets.card.Card
 import org.wycliffeassociates.otter.jvm.app.widgets.card.DefaultStyles
 import org.wycliffeassociates.otter.jvm.app.widgets.card.card
 import tornadofx.*
+import java.util.concurrent.Callable
 
 class ContentGrid : Fragment() {
     private val viewModel: ContentGridViewModel by inject()
 
-    val activeCollection: Property<Collection> = viewModel.activeCollectionProperty
     val activeProject: Property<Collection> = viewModel.activeProjectProperty
+    val activeCollection: Property<Collection> = viewModel.activeCollectionProperty
+    val activeResource: Property<ResourceMetadata> = viewModel.activeResourceProperty
     val activeContent: Property<Content> = viewModel.activeContentProperty
 
+    private val activeResourceIsHelp: BooleanExpression =
+            createBooleanBinding(Callable { activeResource.value.type == "help" }, activeResource)
+
     init {
         importStylesheet<ContentGridStyles>()
         importStylesheet<DefaultStyles>()
         importStylesheet<MainScreenStyles>()
     }
 
+    private fun buildContentWidget(contentGroup: GroupedContent): Node {
+        return when (activeResource.value?.type) {
```

Comment:
I guess you didn't want to check `activeResourceIsHelp` here instead because there will be other types later? (Like tw, which is 'dict')

Reply:
Because I hadn't added isHelp yet when I wrote this line. But your explanation sounds good.